Sources/MongoKitten/CollectionHelpers/Collection+Insert.swift

```diff
@@ -18,7 +18,7 @@ extension MongoCollection {
         }
     }
     
-    public func insertEncoded<E: Encodable>(_ model: E) -> EventLoopFuture<InsertReply> {
```

Comment:
Editing these public function names is not acceptable. This will guaranteed break many APIs.

Reply:
Agree!